### PR TITLE
Testing is a bit easier

### PR DESCRIPTION
**Pull request by veleita**

### CODE REVIEW

File: srcs/server/server.hpp [hunk 1]
```diff
@@ -86,8 +86,8 @@ class	server {
 		std::string get_network_port(void) const 	{return(this->data.network_port);};
 		std::string get_port(void) const 			{return(this->data.port);};
 		std::string get_password(void) const 		{return(this->data.pass);};
-		user& 	get_user(int i);
-		pollfd&	get_pollfd(int i);
+		user& 	get_user(int i)						{return(this->list_of_users.find(i)->second);};
```
Comment (RobertoRobrodri) [inline srcs/server/server.hpp]: I believe there was a reason we had to take them out, but if you tested and it works I´m completely fine

Comment (veleita) [reply srcs/server/server.hpp]: Ok, it took me ages to finally test it but you we're right, it's not compiling. I'll revert the changes.

**Comment by cristina-fb** [comment]
No sé que mierdas estoy aprobando pero ok uwu

**Review by cristina-fb — COMMENTED** [review]
Ahora si que la acepto uwu

**Review by cristina-fb — APPROVED** [review]
QUE uwu

File: tester/server/server.cpp [hunk 2]
```diff
@@ -65,9 +65,6 @@ void	server::init_list_of_cmds(void) // Update with new commands Tested
 	this->list_of_cmds.insert(std::pair<std::string, command_function>("TOPIC", &cmd::topic));
 	this->list_of_cmds.insert(std::pair<std::string, command_function>("INVITE", &cmd::invite));
 	this->list_of_cmds.insert(std::pair<std::string, command_function>("KICK", &cmd::kick));
-	this->list_of_cmds.insert(std::pair<std::string, command_function>("NOTICE", &cmd::notice));
-	this->list_of_cmds.insert(std::pair<std::string, command_function>("PART", &cmd::part));
-	this->list_of_cmds.insert(std::pair<std::string, command_function>("MODE", &cmd::mode));
```
Comment (RobertoRobrodri) [inline tester/server/server.cpp]: Por qué quieres borrar esto?

File: tester/server/server.cpp [hunk 3]
```diff
@@ -241,9 +238,8 @@ void	server::execute_commands(int poll_fd_pos, std::map<std::string, std::string
 void	server::create_channel(user &usr, std::string name) // No test
 {
 	channel cnn(name);
-	usr.set_op(true);
+
 	cnn.add_member(usr);
-	usr.set_op(false);
```
Comment (RobertoRobrodri) [inline tester/server/server.cpp]: Lo mismo aquí

**Review by RobertoRobrodri — CHANGES_REQUESTED** [review]

File: tester/server/server.cpp (hunk 3 above)
Comment (veleita) [reply tester/server/server.cpp]: son errores de no haber hecho pull me parese, corrijo

**Review by RobertoRobrodri — APPROVED** [review]

**Pull request merged**